docs/development/extensions-core/datasketches-tuple.md

```diff
@@ -207,3 +207,39 @@ Returns a human-readable summary of a given ArrayOfDoublesSketch. This is a stri
   "field"  : <post aggregator that refers to an ArrayOfDoublesSketch (fieldAccess or another post aggregator)>
 }
 ```
+
+
+### Constant ArrayOfDoublesSketch 
+
+This post aggregator adds a Base64-encoded constant ArrayOfDoublesSketch value that you can use in other post aggregators.
+```json
+{
+  "type": "arrayOfDoublesSketchConstant",
+  "name": DESTINATION_COLUMN_NAME,
+  "value": CONSTANT_SKETCH_VALUE
+}
+```
+
+### Base64 output of ArrayOfDoublesSketch 
+
+This post aggregator outputs an ArrayOfDoublesSketch as a Base64-encoded string storing the constant tuple sketch value that you can use in other post aggregators. 
+
+```json
+{
+  "type": "arrayOfDoublesSketchToBase64String",
+  "name": DESTINATION_COLUMN_NAME,
+  "field": <post aggregator that refers to a ArrayOfDoublesSketch (fieldAccess or another post aggregator)>
+}
+```
+
+### Estimated metrics values for each column of ArrayOfDoublesSketch
+
+This post aggregator returns a list of estimated values(sum) from a given ArrayOfDoublesSketch. The result is _N_ double values, where _N_ is the number of double values kept in the sketch per key.
```

Comment:
```suggestion
This post aggregator returns a list of estimated values(sum of distincts) from a given ArrayOfDoublesSketch. The result is _N_ double values, where _N_ is the number of double values kept in the sketch per key.
```

Reply:
I think this needs an overall rephrase, basically with each distinct value, there is a `array` of `doubles`, with this post-aggregator we want to provide a double array with sum. 

For E.g.

`Key_1, {1.0, 2.0}`
`Key_2, {2.0, 5.0}`

Output from Post-aggregator is `{3.0, 7.0}`

Reply:
@techdocsmith @vtlim Can you help rephrase this

Reply:
This explanation works. Though it does need to be called out that result is an estimated sum and not accurate sum.

Reply:
could `This post aggregator returns a list of estimated values(sum) from a given ArrayOfDoublesSketch.` be more like `This post aggregator returns a list of estimated sum for each metric value from a given ArrayOfDoublesSketch.` ?

Reply:
Sorry, just saw this. The example isn't clear to me; is the sum being added row-wise or column-wise?

For this example, 
```
Key_1, {1.0, 3.0}
Key_2, {2.0, 5.0}
```

Would the result be `{3.0, 8.0}` or `{4.0, 7.0}`?

Commented a suggestion in https://github.com/apache/druid/pull/13819/files?file-filters%5B%5D=.md&show-viewed-files=true#r1120990146. It would be great to include this example in the docs as well.

Reply:
Result would be `{3.0, 8.0}`

Reply:
> thanks @anshu-makkar. Please let us know once you think it's ready for review. I think we are almost there.

@abhishekagarwal87 : I have added tests for native query as suggested.